omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PatientIdentifierController1_9Test.java

```diff
@@ -114,6 +114,49 @@ public void shouldEditAPatientIdentifier() throws Exception {
 	}
 	
 	@Test
+	public void shouldEditAPatientIdentifierWithPreferredAttibuite() throws Exception {
+		PatientIdentifier patientIdentifier = service.getPatientIdentifierByUuid(getUuid());
+		Boolean originalPreferredStatus = patientIdentifier.getPreferred();
+		Boolean postPreferredStatus = !originalPreferredStatus;
+		SimpleObject postPatientIdentifier = new SimpleObject();
+		postPatientIdentifier.add("preferred", postPreferredStatus);
```

Comment:
Does this test fail before your changes?

Reply:
There was no test for editing preferred. So I added it to test if it could edit preferred

Reply:
In other words, this test is not related to the ticket. Correct?